pkg/commonconfig/commonconfig.go

```diff
@@ -47,7 +47,7 @@ func (cc *CommonConfig) GetProvider() (string, error) {
 		provider = key.ProviderAzure
 
 	default:
-		provider = key.ProviderKVM
+		provider = key.ProviderOpenStack
```

Comment:
I am not sure about this one. Should we dont leave KVM as default and a regexp for OpenStack?

Reply:
hmm I see we don't have a way to know it is OpenStack from the host, right? but how do we deal with GCP or another provider? it seems this is not robust enough

Reply:
Exactly. `kubectl-gs` doesn't support KVM so this felt fine to me for now. We can deal with GCP when we need it. The solution will be coming up with a way to determine the provider of an MC just using a kubeconfig. That could be using athena or maybe a metadata CM that lives in a known place on the MC.